ipatests/test_integration/test_acme.py

```diff
@@ -252,3 +265,35 @@ def test_disable_acme_service(self):
             ['curl', '--fail', self.acme_server],
             ok_returncode=22,
         )
+
+
+class TestACMEwithExternalCA(TestACME):
+    """Test the FreeIPA ACME service with external CA"""
+
+    num_replicas = 0
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        # cache the acme service uri
+        acme_host = f'{IPA_CA_RECORD}.{cls.master.domain.name}'
+        cls.acme_server = f'https://{acme_host}/acme/directory'
```

Comment:
Here's the duplicate code that can be eliminated.

Reply:
This is done. Please have a look.

Reply:
@mrizwan93 all good; thank you for making those changes.